webpack/assets/javascripts/react_app/components/BrowserSupport/BrowserSupport.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'patternfly-react';
+import { sprintf, translate as __ } from '../../common/I18n';
+import { noop } from '../../common/helpers';
+import './BrowserSupport.scss';
+
+const BrowserSupport = ({ show, browserName, initializeBanner }) => {
+  useEffect(() => {
+    initializeBanner();
+  });
+
+  if (!show) return null;
+
+  return (
+    <div className="buorg">
+      <b className="buorg-mainmsg">
```

Comment:
I don't see `buorg-mainmsg` defined in this PR. The same for couple of other classes below.

Reply:
It's defined in the package used for the browser detection.
You got a point we don't need to use them though as we designed the banner from scratch completely.